scripts/asr_language_modeling/ngram_lm/opengrm.py

```diff
@@ -0,0 +1,224 @@
+import argparse
+import os
+import subprocess
+import sys
+
+from nemo.collections.asr.parts.submodules.ctc_beam_decoding import DEFAULT_TOKEN_OFFSET
+
+# Example
```

Comment:
Would you please add some description on what this file do, the parameters, and how they can run it? YOu may add them here at the top of the file.

Reply:
Done